UPGRADE.md

```diff
@@ -1,5 +1,19 @@
 # Upgrade
 
+## unreleased
+
+### Add RestRoutingBundle
+
+To make the update to symfony 5 as seamless as possible a new bundle need to be registered in your `config/bundles.php`:
+
+```php
+HandcraftedInTheAlps\RestRoutingBundle\RestRoutingBundle::class => ['all' => true],
```

Comment:
Many people use 'ClassResourceInterface' to rename routes, perhaps it would be interesting to add it as deprecated, even if the alias exists in the new interface.

Reply:
I meant RouteResource annotation

Reply:
Yeah, I guess that would make sense.

Reply:
This class is not in scope of Sulu when he wants to upgrade FOSRestBundle to 3.0 he need to read the UPGRADE of FOSRestBundle. If its there not clear you maybe should create an issue there. In case of sulu the `RouteResource` is not deprecated as we use the `RestRoutingBundle` which provides this class and so this should work without any problems.